README.md

```diff
@@ -16,13 +16,14 @@ Inclusive Mart is an e-commerce platform for marvel fans, Get Your Best Deal.
 
 ## Contains
 
-Currently, it contains screens for the following page
-- Home (https://inclusive-mart.netlify.app/)
-- Products (https://inclusive-mart.netlify.app/pages/products.html)
-- Cart Management (https://inclusive-mart.netlify.app/pages/cart.html)
-- Wishlist Management (https://inclusive-mart.netlify.app/pages/wishlist.html)
-- Log In (https://inclusive-mart.netlify.app/pages/signin.html)
-- Sign Up (https://inclusive-mart.netlify.app/pages/signup.html)
+Currently, it contains screens for the following pages
+
+- Home (<a href="https://inclusive-mart.netlify.app/" target="_blank">Inclusive Mart Home</a>)
+- Products (<a href="https://inclusive-mart.netlify.app/pages/products.html" target="_blank">Inclusive Mart Products</a>)
+- Cart Management (<a href="https://inclusive-mart.netlify.app/pages/cart.html" target="_blank">Inclusive Mart Cart</a>)
```

Comment:
target="_blank" is not supported in github markdown, so you can remove it and instead use the markdown syntax for links.

Like this -
![image](https://user-images.githubusercontent.com/56336326/154903851-6ebb9407-fe3c-46df-b5ad-ed2164530e35.png)

Reply:
Oh! okay. Thanks for letting me know.